src/components/Home.js

```diff
@@ -0,0 +1,42 @@
+function Home() {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <>
+      <div className="container">
+        <div className="booksContainer">
+          <ul className="booksul">
+            <li>
+              Darkest Hour
+              <span>-Arthur</span>
+            </li>
+            <li>
+              Darkest Hour
+              <span>-Ochieng</span>
+            </li>
+            <li>
+              Climatic diversity
+              <span>-Heldricks</span>
+            </li>
+          </ul>
+        </div>
+        <div className="booksForm">
+          <form onSubmit={handleSubmit}>
+            <label htmlFor="title">
+              Title :
+              <input type="text" name="title" id="title" />
+            </label>
+            <label htmlFor="author">
+              Author :
+              <input type="text" name="author" id="author" />
+            </label>
+            <button type="button">Add Book</button>
+          </form>
```

Comment:
- [ ] I would recommend you create a new component for `BooksForm`. This way it would be easy to maintain the components and content inside them.

Reply:
@Tufahel , sure enough, the more the components the better maintainability 😉. I have implemented that and thank you so much